clap_derive/examples/enum_tuple.rs

```diff
@@ -1,13 +1,13 @@
 //! How to extract subcommands' args into external structs.
 
-use clap::Clap;
+use clap::{Clap, Subcommand};
 
 #[derive(Debug, Clap)]
 pub struct Foo {
     pub bar: Option<String>,
 }
 
-#[derive(Debug, Clap)]
+#[derive(Debug, Clap, Subcommand)]
 pub enum Command {
```

Comment:
Why not just deriving subcommand automatically when Clap is derived on an enum?

Reply:
Too much magic in my opinion. Clap is supposed to be `IntoApp + FromArgMatches`, it's unrelated to `Subcommand`. But technically, we can do that, yes.

Reply:
I'm open to making `#[derive(Clap)]` synonym for `#[derive(IntoApp, FromArgMatches)]` on structs and `#[derive(IntoApp, FromArgMatches, Subcommand)]` on enums. What do you think?